packages/neos-ui/src/Containers/LeftSideBar/NodeTree/index.js

```diff
@@ -123,6 +146,14 @@ export default class NodeTree extends PureComponent {
 
         return (
             <Tree className={classNames}>
+                <a
+                    role="button"
+                    onClick={this.handleCollapseAll}
+                    className={style.collapseAll}
+                    title="Collapse All"
```

Comment:
We'll need i18n for the title. This is a bit tricky,  because the Tree requires an instance of the `i18nRegistry`. 

In this file, registry injection is done via the function `withNodeTypeRegistry` - which needs to be renamed then.

Do you know how to add a translation label from that point onward, or would you like some help with that?

Reply:
I added the i18nRegistry but I am not getting the expected results. For testing purposes I only added de and en translations but instead of "Collapse All", the title becomes "collapseAll" (The ID I gave the new translation).